Wrong copyright header in RetryRule

The build throws an error related to wrong copyright header in `test-e2e/src/test/java/com/sap/jma/utils/RetryRule.java` file. The header in this file contains `2020` and it looks like the copyright header check rule expects `2017` (changing `2020` to `2017` solves the problem).

```
Missing header in: test-e2e/src/test/java/com/sap/jma/utils/RetryRule.java

> Task :test-e2e:licenseTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test-e2e:licenseTest'.
> License violations were found: /Users/[...]/Projects/SAP/java-memory-assistant/test-e2e/src/test/java/com/sap/jma/utils/RetryRule.java}

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.8.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 1m 0s
47 actionable tasks: 41 executed, 6 up-to-date
```

--- a/README.md
+++ b/README.md
@@ -343,6 +343,12 @@ For building behind a proxy, consider setting the proxy-variables as follows:
  ```
 where, for example `[proxy_hostname]` is `proxy.wdf.sap.corp` and `[proxy_port]` is `8080`.
 
+When adding new code files to the project the build might fail with `Missing header in: [...]`. Run the following command to add a license header to all new files:
+
+```
+./gradlew LicenseFormat
+```
+
 ## Running the Java Memory Assistant on Java 11
 
 Due to the restrictions introduced by the Java module system with Java 9, the following additional argument must be passed to the `java` command:

--- a/build.gradle
+++ b/build.gradle
@@ -34,6 +34,7 @@ allprojects {
             gradle = 'SLASHSTAR_STYLE'
         }
         ext.year = Calendar.getInstance().get(Calendar.YEAR)
+        skipExistingHeaders true
     }
 
     group = 'com.sap.jma'

--- a/quality/licensing/HEADER
+++ b/quality/licensing/HEADER
@@ -1,3 +1,3 @@
-Copyright (c) 2017 SAP SE or an SAP affiliate company. All rights reserved.
+Copyright (c) ${year} SAP SE or an SAP affiliate company. All rights reserved.
 This file is licensed under the Apache Software License, v. 2 except as noted
 otherwise in the LICENSE file at the root of the repository.

